src/decimal.rs

```diff
@@ -19,6 +19,8 @@ use diesel::sql_types::Numeric;
 #[cfg(not(feature = "std"))]
 use num_traits::float::FloatCore;
 use num_traits::{FromPrimitive, Num, One, Signed, ToPrimitive, Zero};
+#[cfg(feature = "borsh")]
+use borsh::{BorshDeserialize, BorshSerialize};
```

Comment:
Small nit: you could just add the qualifier on line 102 and get rid of this.
i.e.
```
#[cfg_attr(feature = "borsh", derive(borsh::BorshDeserialize, borsh::BorshSerialize))]
```